### PR TITLE
chore: ✨ add glossary path and set pop-up settings

### LINKED ISSUE #66: Include a glossary with all jargon terms
To help users, it would be good to have a glossary.

### PR DESCRIPTION
**Pull request by signekb**
## Description

This PR adds a glossary page with a table using the [Debruine Quarto-glossary](https://debruine.github.io/quarto-glossary/) extension added in seedcase-project/seedcase-theme#70.

The page looks like this locally: 

image

Closes seedcase-project/seedcase-theme#66

This PR needs an in-depth review.

## Checklist

- [X] Build (website) passed locally - with setup from seedcase-project/seedcase-theme#70
- [X] Updated documentation

### CODE REVIEW

**Review by martonvago — APPROVED** [review]
Looks really cool!!

File: docs/guide/packages.qmd [hunk 1]
```diff
@@ -4,10 +4,7 @@ order: 1
 jupyter: python3
 ---
 
-At the core of Sprout is the [data
-package](/docs/design/implementation/outputs.qmd), which is a
-standardized way of structuring and sharing data. This guide will show
-you how to create and manage data packages using Sprout.
+At the core of Sprout is the [{{< glossary "data package">}}](../glossary.qmd), which is a standardized way of structuring and sharing data. This guide will show you how to create and manage data packages using Sprout.
```
Comment (lwjohnst86) [inline docs/guide/packages.qmd:7]: Hmm, we might be able to write our own shortcode for this! But let's low priority and we'll do it later.

File: docs/glossary.qmd [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Glossary"
+---
+
+::: hidden
+{{< glossary "data package" >}}
```
Comment (lwjohnst86) [inline docs/glossary.qmd:6]: Unfortunate that we have to type it out twice. But low priority and we can figure it out later.

**Review by lwjohnst86 — APPROVED** [review]
super nice!!

File: _quarto.yml [hunk 3]
```diff
@@ -5,6 +5,11 @@ project:
     - "docs/*"
     - "index.qmd"
 
+glossary:
+  path: _extensions/debruine/glossary/glossary.yml
```
Comment (lwjohnst86) [inline _quarto.yml:9]: Oops, didn't notice this, but this should be in our theme, not theirs.

File: docs/glossary.qmd (hunk 2 above)
Comment (signekb) [reply docs/glossary.qmd:6]: Agree! I very much don't like it. Also, if we forget to add one here, it won't show up in the table.

**Pull request merged**